### PR TITLE
Updated README.md to include all installation instructions

### PR DESCRIPTION
**Pull request by oz03-hub**

### CODE REVIEW

**Review by SergePogorelov — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -26,7 +26,94 @@ Estimating the geographical range of a species from sparse observations is a cha
 ```
 
 #### Data Download and Preparation
-Instructions for downloading the data in `data/README.md`.
+# Instructions for Data Preparation
+After following these instructions, the `data` directory should have the following structure:
```
Comment (angelazhu2) [inline README.md]: this directory explanation section is not readable, can we move it to after the instructions of how to setup and make it clear it's just an explanation of how the folder is structured. 

If we are going to put all the readmes together, we should try to make them readable for the next user.

File: README.md [hunk 2]
```diff
@@ -26,7 +26,94 @@ Estimating the geographical range of a species from sparse observations is a cha
 ```
 
 #### Data Download and Preparation
-Instructions for downloading the data in `data/README.md`.
+# Instructions for Data Preparation
```
Comment (cheerstopaula) [inline README.md]: There is a problem with the headers here and in general. If you see the README on the GUI, it is hard to follow the data section and below as well (You can see it by opening your branch on the web view). Make sure the headers' style are consistent and there aren't duplicate titles (e.g. "Data Download and Preparation" and "Instructions for Data Preparation").

**Review by cheerstopaula — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -26,7 +26,94 @@ Estimating the geographical range of a species from sparse observations is a cha
 ```
 
 #### Data Download and Preparation
```
Comment (angelazhu2) [inline README.md]: Please add titles for the two sections so there's "set up instructions" and "run the code" (or something like that), since we are making the readme file longer let's make it easier to navigate.

**Review by angelazhu2 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (oz03-hub) [reply README.md]: pushed new changes

File: README.md (hunk 1 above)
Comment (oz03-hub) [reply README.md]: new changes should fixed it

File: README.md (hunk 3 above)
Comment (oz03-hub) [reply README.md]: new changes

**Comment by oz03-hub** [comment]
Take a look at the new README.md it should be as wanted, specifically go to https://github.com/UMassCDS/inaturalist-sinr/tree/oz03-hub-add-installation-instructions

File: README.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ Estimating the geographical range of a species from sparse observations is a cha
 ![Model Prediction](images/sinr_traverse.gif)
```
Comment (angelazhu2) [inline README.md:9]: I think you need to pull from main, cause there's a title that makes it clear it's our forked repository

**Review by angelazhu2 — APPROVED** [review]

**Review by cheerstopaula — APPROVED** [review]
